modules/schema/src/main/resources/lucuma/odb/graphql/OdbSchema.graphql

```diff
@@ -10196,6 +10223,11 @@ type Target {
   """
   name: NonEmptyString!
 
+  """
+  calibration role
+  """
+  calibrationRole: CalibrationRole
```

Comment:
By specifying this as a `calibration role` rather than just a generic `role`, are we fairly certain that we won't have any other types of roles for targets in the future? I guess if we do, we could relax the name in the future.

Reply:
It is possible I wanted to use this specifically for the `CalibrationRole` enumeration